aodntools/timeseries_products/velocity_hourly_timeseries.py

```diff
@@ -121,6 +122,8 @@ def velocity_hourly_aggregated(files_to_agg, site_code, input_dir='', output_dir
     ## remove bad files form the list
     for file in bad_files.keys():
         files_to_agg.remove(file)
+    if len(files_to_agg) == 0:
```

Comment:
repeating pattern - factor out that file_to_agg removal/checker into a function.

Reply:
Will do, later, in a refactoring PR.